### ISSUE TITLE
Difference in file size between .omap/.ocd

### ISSUE THREAD

**Issue by SwStef** [OpenOrienteering/mapper#1344]
Hi, I´m working on a rather big map using vers. 0.8.4 and found some annoying issues. 
1. When using .omap, as I have to to be able to use it fully on my android device, it´s appr. 6500kB, but when saving as .ocd it`s just appr. 3400kB. 
So when exporting to PDF or PNG for printing or for livelox/route gadget I have to use the .ocd to get a manageable size. This means that I have to change between .omap and .ocd for different operations. 
2. Then there is a problem when using the "cut out" tool to make a smaller map-part/section. When using .omap, the size of the smaller part/section still is 6500kB or gets even bigger(!?). When using .ocd the same small part/section has the correct size 300kB. 
Why is that?

### Configuration
Mapper Version:0.8.4
Operating System: win 10 pro, android

**Comment by ollesmaps** [comment]
Have you tried Edit / Clear Undo Redo history ?

**Comment by SwStef** [comment]
No. I must have missed that one. How/where do I do it?

**Comment by lpechacek** [comment]
In desktop version (i.e. not on Android) choose **Edit** menu and the last item is **Clear undo / redo history**. Mapper keeps undo information in the map file and it is possible to undo steps done in past editing session(s). HTH

**Comment by SwStef** [comment]
Thanks:),
It solved the problem with the "cut-out-section". 
The file size of the whole map lost just appr. 100kB, so the difference there remains.

**Comment by jmacura** [comment]
There will be always a difference in size as .omap and .ocd are different file formats. That meaning that they represent the same data in a different form, internally. There is no chance for them to be of equal size, not even for empty map.
And as .omap is based on XML format, which is a bit wordy, it tend to be generally larger then the binary representation in .ocd.

**Comment by SwStef** [comment]
Many thanks for a good explanation:)!
